src/Components/Forms/src/DataAnnotationsValidator.cs

```diff
@@ -12,6 +12,7 @@ public class DataAnnotationsValidator : ComponentBase, IDisposable
     private EditContext? _originalEditContext;
 
     [CascadingParameter] EditContext? CurrentEditContext { get; set; }
+    [Inject] IServiceProvider? ServiceProvider { get; set; }
```

Comment:
```suggestion

    [Inject] private IServiceProvider? ServiceProvider { get; set; }
```

I'm going to eventually turn on the rule to specify accessibility in this repo, so might as well get it right the first time around.